src/python/pants/backend/python/tasks2.py

```diff
@@ -0,0 +1,24 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+from pants.backend.python.tasks import (GatherSources, PytestPrep, PytestRun, PythonBinaryCreate,
```

Comment:
I'm noticing now that this is broken:

A) These imports are all invalid, as those classes are defined in modules inside `tasks`, not in `tasks` itself.

B) Even if we fix that, any existing import statements that try to import from modules in `tasks2` will now fail, instead of showing the deprecation message but succeeding. 

I see no alternative but to restore the modules in `tasks2` and have each one just import from the sibling module in `tasks` and show a deprecation message.

Or am I missing something? This definitely caused pants to fail when I tried to register a `tasks2` task.

Reply:
...hm, of course. sheesh... not sure how I missed that.

Reply:
I'll tackle that today.

Reply:
https://github.com/pantsbuild/pants/pull/5421